server.js

```diff
@@ -15,16 +15,14 @@ app.use(bodyParser.json());
 
 mongoose.connect('mongodb://localhost/Cinema', function(err) {
   if (err) {
-    res.status(500).json({
-      message: 'Нет подключения к Базе данных'
-    });
+   return res.status(500);
```

Comment:
I meant that you should use some generic message for all 500 errors, not to remove it completely